INSTALL

```diff
@@ -394,6 +394,11 @@ MFEM_USE_AMGX = YES/NO
    Allows the user to use SparseMatrices and HypreParMatrices to solve linear
    systems with the routines from the AmgX library.
 
+MFEM_USE_MAGMA = YES/NO
```

Comment:
There's another section in this file where we should add MAGMA, e.g. search for PETSC to see where it is mentioned here.

Reply:
Added in 7b35f3cfde91dae3145ece73af563828bdc30933.